webapp/application.py

```diff
@@ -18,7 +18,7 @@
     "27992": "I've decided to stay with my current employer",
     "36714": "I cannot complete the assessment",
     "35818": "The position isn't a good fit",
-    "33": "Other"
+    "33": "Other",
```

Comment:
This comma is mistakenly added. I will remove it.

Reply:
putting a comma at the end of a list of dictionary items, or arguments, or whatever doesn't matter in Python. You can either add the trailing comma or not. Adding it makes it easier to reorder the items without causing a syntax error. Generally we should just let Black take care of this, I *think* bash tends to add these commas by default. But it really doesn't matter which we do.